src/snowflake/snowpark/functions.py

```diff
@@ -3837,7 +3837,7 @@ def array_remove_nulls(col: Column) -> Column:
         ),
         separator=lit(sep, _emit_ast=False),
         _emit_ast=False,
-    ).alias(f"CONCAT_WS_IGNORE_NULLS('{sep}', {names})", _emit_ast=False)
+    )._alias(f"CONCAT_WS_IGNORE_NULLS('{sep}', {names})")
```

Comment:
what's the criteria to use this internal `_alias`?
I'm thinking in the future, if other developers are implementing new functions or other functionality which uses internal alias:

1. how can they be aware of this _alias
1. how they know they should use _alias vs alias